cross_compile.md

```diff
@@ -0,0 +1,82 @@
+# cross compile grpc-rs to windows under *nix
+
+### Fisrt you need to install mingw
+
+```bash
+# MacOS
+brew install mingw-w64
+
+# CentOS
+yum install mingw64-openssl-static mingw64-zlib-static mingw64-winpthreads-static
+```
+
+### fix cmake
+
+```
+# modify grpc-rs/grpc-sys/build.rs
+# fix SYSTEM_PROCESSOR
+"CMAKE_SYSTEM_PROCESSOR", get_env("CARGO_CFG_TARGET_ARCH").unwrap()
+# fix try_run
+"CMAKE_CROSSCOMPILING", "true"
+```
+
+#### all diff in fn build_grpc
+
+```rust
+    let dst = {
+        let mut config = Config::new("grpc");
+        if get_env("CARGO_CFG_TARGET_OS").map_or(false, |s| s == "macos") {
+            config.cxxflag("-stdlib=libc++");
+        }
+        config
+            .define("CMAKE_SYSTEM_PROCESSOR", get_env("CARGO_CFG_TARGET_ARCH").unwrap())
+            .define("CMAKE_CROSSCOMPILING", "true")
+            .build_target(library)
+            .uses_cxx11()
+            .build()
+        // config.build_target(library).uses_cxx11().build()
+    };
+```
+
+#### fix find zlib
```

Comment:
"#### fix" -> "### Fix"

or **"## Fix find zlib"** if it is the same level with "Fix CMake"